cookbook/web-server/nginx.rst

```diff
@@ -32,7 +32,7 @@ The Nginx configuration could look something like.
           access_log off;
           expires 1y;
           add_header Pragma public;
-          add_header Cache-Control "public";
+          add_header Cache-Control "public, immutable";
```

Comment:
@alexander-schranz Does this rule also apply for the images placed e.g. in `public/website/images/logo.png`? Because this might change and should not use the `immutable` flag i guess

Reply:
They should also be versionated as they already have a cachelifetime from 1year.

Reply:
okay then